Source/UIKit additions/SVGKFastImageView.m

```diff
@@ -74,7 +78,12 @@ -(id)initWithFrame:(CGRect)frame
 	self = [super initWithFrame:frame];
 	if( self )
 	{
+#if SVGKIT_UIKIT
```

Comment:
What's the different between #if on SVGKIT_UIKIT and #if MAC_OS ?

Reply:
Current seems have the same meaning. Because we support only iOS/tvOS/macOS. `SVGKIT_UIKIT` is standfor iOS or tvOS. So `#if !SVGKIT_UIKIT` is actually means macOS.

However, maybe someone in the future want to add watchOS support ? This assume will fail.

Reply:
This patch uses both, and unless there's a strong reason for that, I think it should only use one style or the other.

Does watchOS have UIKit? Or is it an incompatible UIKit?

I like the idea of using a SVGKIT_UIKIT everywhere, as that seems to be accurate (the problem isn't "OSX" so much as "OSX currently doesn't have a UIKit implemetnation built-in", right)?

Reply:
Correct. OS X use AppKit, iOS / tvOS use UIKit. At least now. Apple may release a real UIKit for macOS in the future (next year), but however, there are no API guideline for that so current we have no way to know how to detect.

WatchOS contains a subset of UIKit, actually the `UIKitCore.framework`, the only public class are `UIImage`, `UIFont`, `UIColor`. All the UIView/UIViewController class is not available for watchOS.